io.github.fkinoshita.Wildcard.json

```diff
@@ -0,0 +1,63 @@
+{
+    "id" : "io.github.fkinoshita.Wildcard",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "wildcard",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--env=RUST_LOG=wildcard=debug"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
This is not allowed

Reply:
Thanks!

Reply:
you'll need to either vendor your crates in a tarball, or to use https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo to generate a manifest.

Reply:
You totally ignored this comment. This is why it's not building. No amount of retrying will work, the builds are sandboxed without network access.

Reply:
Thank you, I didn't saw your comment before.

Sorry for not being an expert on how to build flatpaks.